docs/Developers/Review.rst

```diff
@@ -65,6 +65,15 @@ The review process for coala is as follows:
    ``corobo mark wip <URL>`` or ``corobo mark pending <URL>``.
 3. The developers will acknowledge the commits by writing
 
+   * In case a member is reviewing it:
+
+    - ``Looks good to me`` better known as ``LGTM`` in case the commit is ready
+         to be merged,or
+    - Additional comments if something needs to be rectified.
+
```

Comment:
Only one blank line would be sufficient here.

Reply:
Done!

Reply:
@kriti21 : why removal of a new line is causing the build fail?